timezone/src/modules/Timezone.rb

```diff
@@ -984,6 +986,23 @@ def Summary
       HTML.List(ret)
     end
 
+
+    # Checks whether the system has Windows installed on a primary partition with NTFS
+    def system_has_windows?
+      storage = Y2Storage::StorageManager.instance
+      Storage::Ntfs::all(storage.probed).each do |ntfs|
+        ntfs.blk_devices.each do |blk_device|
+          if Storage::partition?(blk_device)
+            if Storage::to_partition(blk_device).type == Storage::PartitionType_PRIMARY
+              return true if ntfs.detect_content_info.windows?
+            end
+          end
+        end
```

Comment:
For some reasons (explained in the other pull request), I would like to merge this https://github.com/yast/yast-storage-ng/pull/92 and change the code to something like
```ruby
def system_has_windows?
  !disk_analyzer.windows_partitions.empty?
end

def disk_analyzer
  @disk_analyzer ||= begin
    devicegraph = Y2Storage::StorageManager.instance.probed
    Y2Storage::DiskAnalyzer(devicegraph)
  end
end
```